### ISSUE TITLE
OpenSSL should not take over "man passwd"

### ISSUE THREAD

**Issue by chkno** [NixOS/nixpkgs#69443]
**Describe the bug**
`man passwd` should describe the password-changing utility that you get when you type `passswd` in a shell.  Instead, it describes a rarely-used sub-command of openssl.

**To Reproduce**
1. `man passwd`

**Expected behavior**
I see documentation about changing user passwords in /etc/shadow, like http://man7.org/linux/man-pages/man1/passwd.1.html .

**Additional context**
Here is the list of openssl subcommands that collide with actual commands (according to the command-not-found perl script):
```
ec      eplot
passwd  busybox su
ts      ts moreutils
version mailman
sha1    hashcash
base64  busybox coreutils coreutils-full libb64
```
I notice that I don't have this problem with base64 -- `man base64` brings up the coreutils manual as expected.

I am able to view the passwd manpage directly out of the store with `man /nix/store/*-shadow-*-man/share/man/man1/passwd.1.gz`

**Metadata**
```
$ nix run nixpkgs.nix-info -c nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.75, NixOS, 19.03.173547.2c61951df92 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.03.173547.2c61951df92, nixos-hardware, nixos-unstable-20.03pre193829.f0fec244ca3"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by matthewbauer** [comment]
It actually looks like we already have a workaround for this! It must have stopped working at some point:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/openssl/default.nix#L78-L81

**Comment by chkno** [comment]
That, or it was fixed at master and hadn't reached a release branch yet (and I didn't yet know how to check master).

Looks like this was fixed in 92b96ce63fa8752d0487edbe026a37ff765b33b3.

I can no longer reproduce this problem.
